Description: Fix FTBFS for handling clutter_init() return value
 Proper handling return value of clutter_init()
Author: Ying-Chun Liu (PaulLiu) <paulliu@debian.org>
Bug-Debian: http://bugs.debian.org/646445
Last-Update: 2011-11-12
Index: clutter-gesture-0.0.2.1/tests/test-events.c
===================================================================
--- clutter-gesture-0.0.2.1.orig/tests/test-events.c	2010-01-06 12:11:44.000000000 +0800
+++ clutter-gesture-0.0.2.1/tests/test-events.c	2011-11-13 00:01:11.739835870 +0800
@@ -272,12 +272,16 @@
 		  /* ycol = { 0xff, 0xff, 0, 0xff }; */
 		  /*ncol = { 0, 0, 0, 0xff }; */
   ClutterGesture *gesture = NULL;
+  ClutterInitError clutter_err;
 
   g_type_init();
 
   clutter_x11_enable_xinput();
 
-  clutter_init (&argc, &argv);
+  clutter_err = clutter_init (&argc, &argv);
+  if (clutter_err != CLUTTER_INIT_SUCCESS) {
+      g_error ("clutter_init() error\n");
+  }
 
 //  clutter_set_motion_events_frequency(120);
 
Index: clutter-gesture-0.0.2.1/tests/test_engine.c
===================================================================
--- clutter-gesture-0.0.2.1.orig/tests/test_engine.c	2010-01-06 12:11:44.000000000 +0800
+++ clutter-gesture-0.0.2.1/tests/test_engine.c	2011-11-13 00:07:32.133794928 +0800
@@ -242,10 +242,12 @@
 		  ycol = { 0xff, 0xff, 0, 0xff },
 		  ncol = { 0, 0, 0, 0xff };
   ClutterGesture *gesture = NULL;
+  ClutterInitError clutter_err;
 
-  clutter_init (&argc, &argv);
-
-
+  clutter_err = clutter_init (&argc, &argv);
+  if (clutter_err != CLUTTER_INIT_SUCCESS) {
+    g_error ("clutter_init() error\n");
+  }
 
   stage = clutter_stage_get_default ();
   g_signal_connect (stage, "event", G_CALLBACK (input_cb), "stage");
